Robots/goalkeeper_controller.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import math
+from GameEngine.ball_model import BallModel
+from GameEngine.robot_model import RobotModel
+from Robots.robot_control import Robot, Goal
+
+
+class GoalkeeperController(Robot):
+
+    def __init__(self):
+        self._goalkeeper = None
+        self._ball = None
+        self._lookhead = 10
+
+    def get_action(self, goal) -> tuple:
+        if goal == Goal.ChaseBall:
+            lookahead = self._ball.get_position()
+
```

Comment:
line breaks should be between different blocks of the code, between methods in class, double lines between imports and other code and after the class 

Take a look on PEP8 -> for example here: [PEP8 RealPython](https://realpython.com/python-pep8/)